server/backend/quiz/models.py

```
@@ -0,0 +1,51 @@
+from django.db import models
+from django.contrib.auth.models import User
```

Ruff (0.8.2)

`django.contrib.auth.models.User` imported but unused

Remove unused import: `django.contrib.auth.models.User`

(F401)